client/src/actions.js

```diff
@@ -0,0 +1,4 @@
+import * as types from './constants/ActionTypes'
+
+export const userLogin = id => ({ type: types.USER_LOGIN, id })
+export const userLogout = () => ({ type: types.USER_LOGOUT })
```

Comment:
Should we break this out into a specific file under an actions directory? Like `client/src/actions/login.js`? We dont necessarily have to since this app isn't going to be giant but I'm just thinking ahead to the other actions to come